libctru/source/internal.h

```diff
@@ -8,6 +8,12 @@
 #define THREADVARS_MAGIC  0x21545624 // !TV$
 #define FS_OVERRIDE_MAGIC 0x21465324 // !FS$
 
+extern const size_t __tdata_align;
```

Comment:
I think this could be a weak symbol if backwards compatibility with older versions of `devkitarm-crtls` is needed, which could fall back to the old behavior (align of 8).

Reply:
Backwards compatibility is not a concern. libctru and devkitarm-crtls will be updated in lockstep.